src/api/app/views/webui/users/notifications/_notification_filter.html.haml

```diff
@@ -56,6 +56,15 @@
                                                             label_icon: 'fas fa-people-group',
                                                             checked: selected_filter[:kind]&.include?('member_on_groups') }
 
+  .mt-4.mb-4
+    %h6.px-3.py-2
+      %b Reports
+    .px-3
+      = render partial: 'webui/shared/select', locals: { filter_item: 'report_with_decision',
```

Comment:
@ncounter I would say we should also check for ` policy(Report.new).notify?` to render this only to the users who are in beta and actually use it

Reply:
Good point, I added that :+1: